po/terminator.pot

```diff
@@ -1,1776 +0,0 @@
-# SOME DESCRIPTIVE TITLE.
```

Comment:
Why deleting the POT? Shouldn't we keep it for translators?

Reply:
Hmm, definitely shouldn't have deleted that.

Reply:
Looks like you brought it back in a following commit.

Reply:
Yes, checkout, commit, update. :wink: